package.json

```diff
@@ -1,20 +1,27 @@
 {
-  "name": "react-redux-group-project",
+  "name": "my-app",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@jest/globals": "^29.6.4",
     "@reduxjs/toolkit": "^1.9.5",
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "axios": "^1.5.0",
+    "axios": "^0.21.1",
+    "eslint-config-airbnb": "^19.0.4",
+    "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^4.10.1",
     "react-redux": "^8.1.2",
     "react-router-dom": "^6.15.0",
     "react-scripts": "5.0.1",
     "redux": "^4.2.1",
+    "redux-devtools-extension": "^2.13.9",
     "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.4.2",
+    "styled-components": "^6.0.7",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
- You have done great building out your capstone :+1: But there are a few changes and requirements you need to complete before moving to the next milestone. :smile: 

- [ ] Looking at the video below, you can see that when I run `npm start`, your application starts but when it opens in the browser, the webpage is white and blank. It shows an error in the console. I'll suggest you fix this and make sure your project is running properly before requesting for review :+1: :100: :+1: 

- - Video reference:

https://github.com/Nessrine88/React-redux-Group-Project/assets/107632780/86a0e44d-0cea-440d-b2cb-2781931aeb61

- Your project needs to be running properly so that it can be properly reviewed and also so that it can be accessible to users and your potential employer :+1: :wave: